src/server.js

```diff
@@ -354,3 +367,76 @@ function auth(data) {
 		}
 	}
 }
+
+
+function IterateOver(list, iterator, callback) {
```

Comment:
I would use promises, if its allowed. I remember something about avoiding them in lounge, so i made chunky workaround. If it is allowed, a example would be nice.